webrtc/methods/participants/createParticipant.md

```diff
@@ -25,8 +25,9 @@ You should not include sensitive or personally-identifiable information in any t
 | Parameter                   | Description                                                                                       
 |:----------------------------|:--------------------------------------------------------------------------------------------------
 | callbackUrl                 | Full callback url to use for notifications about this participant                                 
-| publishPermissions          | Defines the types of media this participant can publish. Accepted values: `AUDIO`,`VIDEO`                                                                                           
-| tag                         | User defined tag to associate with the participant                                                
+| publishPermissions          | Defines the types of media this participant can publish. Accepted values: `AUDIO`,`VIDEO`
+| tag                         | User defined tag to associate with the participant
+| deviceApiVersion            | Optional parameter that sets the version of the signaling API the participant will use. Accepted values: `V2`,`V3`. Defaults to `V2`
```

Comment:
can it be upper or lower case or does it matter?

Reply:
It doesn't matter